pkg/translate/translate_value.go

```diff
@@ -80,6 +80,9 @@ var (
 	// Component enablement mapping. Ex "{{.ValueComponent}}.enabled": {"{{.FeatureName}}.Components.{{.ComponentName}}.enabled}", nil},
 	// Feature enablement mapping. Ex: "{{.ValueComponent}}.enabled": {"{{.FeatureName}}.enabled}", nil},
 	componentEnablementPattern = "{{.FeatureName}}.Components.{{.ComponentName}}.Enabled"
+	// specialComponentPath lists cases of component path of values.yaml we need to have special treatment.
+	specialComponentPath = []string{"mixer.policy", "mixer.telemetry", "gateways.istio-ingressgateway", "gateways.istio-egressgateway",
```

Comment:
It's more idiomatic to use a map for this so you can avoid the for loops below and just do specialComponentPath[key] to test.

Reply:
done